docs/layers/identity/aws-sso.mdx

```diff
@@ -112,45 +114,75 @@ For providers not included in the following section, please [follow the AWS docu
 
   </TabItem>
 
-  <TabItem value="microsoft365" label="Microsoft 365">
+  <TabItem value="microsoft-entra-id" label="Microsoft Entra ID">
 
     <Intro>
-      Microsoft 365 (formerly known as Office 365) can be used as an identity provider for AWS (Amazon Web Services) through a feature called AWS Single Sign-On (AWS SSO).
+      Microsoft Entra ID (formerly known as Office 365, Microsoft 365, Azure AD) can be used as an identity provider for AWS (Amazon Web Services) through a feature called AWS Single Sign-On (AWS SSO).
     </Intro>
 
-    AWS SSO allows organizations to centralize identity management and provide users with seamless access to AWS resources using their existing Microsoft 365 credentials.
+    AWS SSO allows organizations to centralize identity management and provide users with seamless access to AWS resources using their existing Microsoft Entra ID credentials.
 
-    ### Setup Microsoft 365
+    ### Setup Microsoft Entra ID
 
     <Steps>
-      1. Under [https://aad.portal.azure.com/#allservices/category/All](https://aad.portal.azure.com/#allservices/category/All)  go to **Enterprise Applications** <img src="/assets/refarch/screen-shot-2023-01-13-at-8.33.13-am.png" /><br/>
-
-      2. Click **New Application**
-
-      3. Choose the right application, for SSO it’s **AWS IAM Identity Center (successor to AWS Single Sign-On)**
-        <img src="/assets/refarch/screen-shot-2023-01-13-at-8.36.16-am.png" /><br/>
-
-      4. Click **Create**, default options are fine
-
-      5. On the Left Panel Click **Single sign-on,** then download the XML File by Pressing the Button on step 5 **Set up AWS IAM Identity Center (successor to AWS Single Sign-On**)
+        #### <StepNumber overrideNumber={1}/> Under [Microsoft Entra's Admin Center](https://entra.microsoft.com/) 
```

Comment:
WHy do we need this new `overrideNumber`? The index should be calculated automatically by the `<Steps>` context